src/Component/BlogPage.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import './BlogPage.css'
+
+import BlogMiddleComp from './BlogMiddleComp'
+import { data } from '../data'
+function BlogPage() {
+
+    const [searchTerm, setSearchTerm] = useState('');
+    const handleSearch = (e) => {
+        setSearchTerm(e.target.value)
+        console.log(searchTerm)
+    }
+
+    const searchFilter = () => {
+
+        let arr = data.filter(function (d) {
+            // console.log(d.name)
+            if (d.name != undefined) {
+                return d.name.toLowerCase().includes(searchTerm.toLowerCase())
+            }
+        })
+
+        return arr;
+    }
+
+    return (
+
+        <div className='blogs'>
```

Comment:
Maintain string quotes consistency. Some places, you are using single quotes while at others double. Choose one. Eslint should enforce this.